src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/DropDownButton/DropDownButtonPage.xaml.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Markup;
+using Windows.UI;
+using System.Windows.Input;
+using Uno.UI.Samples.Controls;
+
+namespace MUXControlsTestApp
+{
+
+	[SampleControlInfo("DropDownButton", "MUX Page")]
+	public sealed partial class DropDownButtonPage
+	{
+		private int _clickCount = 0;
+		private int _flyoutOpenedCount = 0;
+		private int _flyoutClosedCount = 0;
+
+		private Flyout _flyout;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make '_flyout' 'readonly'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5101549)

Reply:
@jeromelaban Should I address these or keep it (as it is direct copy from Windows UI tests?)

Reply:
@MartinZikmund In this case you can ignore. The main goal of codacy is to find potential bugs...